packages/sdk-node/package.json

```diff
@@ -1,7 +1,7 @@
 {
+  "name": "@clerk/clerk-sdk-node",
   "version": "4.12.13",
   "license": "MIT",
-  "type": "commonjs",
```

Comment:
We shouldn't define this in a dual CJS-ESM package.

As per https://nodejs.org/api/packages.html#type

> If the nearest parent package.json lacks a "type" field, or contains "type": "commonjs", .js files are treated as [CommonJS](https://nodejs.org/api/modules.html).
> Regardless of the value of the "type" field, .mjs files are always treated as ES modules and .cjs files are always treated as CommonJS.

So we can rely on the fact that we have .mjs files here